01_fib/baijie/fib.go

```diff
@@ -0,0 +1,35 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+// fib function to generate fibonacci sequence
+func fib(i int) {
+	// initial values
```

Comment:
This is positively a comment that can be left out, as in my opinion it does not add value to the code.
```suggestion
```

Reply:
Done